lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -148,11 +172,16 @@ window.runLighthouseInExtension = function(options, requestedAggregations) {
   // Default to 'info' logging level.
   log.setLevel('info');
   const connection = new ExtensionProtocol();
-  return connection.getCurrentTabURL()
+  return enableOtherChromeExtensions(false)
+    .then(_ => connection.getCurrentTabURL())
     .then(url => window.runLighthouseForConnection(connection, url, options, requestedAggregations))
     .then(results => {
+      enableOtherChromeExtensions(true);
```

Comment:
these have to be returned for the rejection to get into the promise chain. 
```js
.then(results => {
  return enableOtherChromeExtensions(true).then(_ => {
    const blobURL = window.createReportPageAsBlob(results, 'extension');
    chrome.tabs.create({url: blobURL});
  });
}).catch(err => {
  return enableOtherChromeExtensions(true).then(_ => {
    throw err;
  });
});
```
maybe?